### PR TITLE
Css luigy

### PR DESCRIPTION
**Pull request by Luigy-Lemon**
Fixed the css for multiple views and components on the **Withdraw Overview page**. 
Added the **collateral Composition feature**

```ts
const collateralComposition = React.useMemo(() => {
		const compositionArray = allReservesData?.map(next => {
			if (next.daiWeiPriceTotal != undefined && next.decimals != undefined && totalCollateralValue != undefined) {
				const decimalPower = BigNumber.from(10).pow(next.decimals);
				return next.daiWeiPriceTotal.mul(decimalPower).div(totalCollateralValue)
			}
			else return BigNumber.from(0);
		});
		return (compositionArray) ? compositionArray.map(share => {
			if (share.gt(0)) {
				return formatEther(share.mul(100))
			}
			else return null
		}) : []
	}, [allReservesData, totalCollateralValue]);

const collateralData = collateralComposition.map((x, index) => {
		if (x != null) return x.substr(0, x.indexOf(".") + 3);
	})

### CODE REVIEW

File: src/views/common/WithdrawDash.tsx [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+			else return null
+		}) : []
+	}, [allReservesData, totalCollateralValue]);
+
+	const collateralData = collateralComposition.map((x, index) => {
+		if (x != null) return x.substr(0, x.indexOf(".") + 3);
+	})
+
+	const [isSmallerThan900] = useMediaQuery("(max-width: 900px)");
+	const [isSmallerThan400] = useMediaQuery("(max-width: 400px)");
```
Comment (zedkai) [inline src/views/common/WithdrawDash.tsx]: Less hooks is more efficient, do so when they can be combined.
```suggestion
	const [isSmallerThan400, isSmallerThan900] = useMediaQuery(["(max-width: 400px)", "(max-width: 900px)"]);
```

File: src/views/common/WithdrawDash.tsx [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+import { useUserDepositAssetBalances } from "../../queries/userAssets";
+import { useUserAccountData } from "../../queries/userAccountData";
+import { useUserReserveAssetBalancesDaiWei } from "../../queries/userAssets";
+import { useProtocolReserveData } from "../../queries/protocolReserveData";
+import { useUserAssetBalance } from "../../queries/userAssets";
+import { fontSizes, spacings, assetColor } from "../../utils/constants";
+import { ModalIcon } from "../../utils/icons";
+import { TokenIcon } from "../../utils/icons";
+import { forEach } from "lodash";
+import { string } from "prop-types";
```
Comment (zedkai) [inline src/views/common/WithdrawDash.tsx]: This probably shouldn't be here.
```suggestion
```

File: src/views/common/WithdrawDash.tsx [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+
+type WithdrawDashProps = {
+	token: ReserveTokenDefinition;
+};
+
+const WithdrawDash: React.FC<WithdrawDashProps> = ({
```
Comment (zedkai) [inline src/views/common/WithdrawDash.tsx]:
```suggestion
export const WithdrawDash: React.FC<WithdrawDashProps> = ({
```

File: src/views/common/WithdrawDash.tsx [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+							</PopoverBody>
+						</PopoverContent>
+					</Popover>
+				</Flex>
+			</Flex>
+		</VStack >
+	);
+};
+
+export default WithdrawDash;
```
Comment (zedkai) [inline src/views/common/WithdrawDash.tsx]: Consider default-exports problematic.
```suggestion
export default WithdrawDash;
```

**Review by zedkai — CHANGES_REQUESTED** [review]

File: src/views/common/WithdrawDash.tsx (hunk 1 above)
Comment (Luigy-Lemon) [reply src/views/common/WithdrawDash.tsx]: ok fixed

File: src/views/common/WithdrawDash.tsx (hunk 2 above)
Comment (Luigy-Lemon) [reply src/views/common/WithdrawDash.tsx]: also removed lodash

File: src/views/common/WithdrawDash.tsx (hunk 3 above)
Comment (Luigy-Lemon) [reply src/views/common/WithdrawDash.tsx]: fixed also on DepositDash

File: src/views/common/WithdrawDash.tsx (hunk 4 above)
Comment (Luigy-Lemon) [reply src/views/common/WithdrawDash.tsx]: Fixed also on DepositDash

**Review by zedkai — APPROVED** [review]

**Pull request merged**